src/doc/src/reference/workspaces.md

```diff
@@ -68,8 +68,13 @@ you want to keep all the packages organized in separate directories.
 # [PROJECT_DIR]/Cargo.toml
 [workspace]
 members = ["hello_world"]
+resolver = "2"
 ```
 
+The [`resolver`] key here effects how a package that is depended on in multiple
+ways with different features are resolved. Some libraries depend on it being
+set to 2.
```

Comment:
I'd recommend shifting the focus to fit within this section.  We delegate explanation of the field to the resolver page.

Something like

```md
Without a root package to fall back on, the [`resolver`] field should be set in the `[workspace]` table.
```
(at first I was worried this was too strong of a recommendation but it is roughly what the `resolver` page says)